package.json

```diff
@@ -38,13 +39,7 @@
     "eslint-plugin-import": "^2.19.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-no-only-tests": "^2.4.0",
-    "husky": "^3.1.0",
-    "jest": "^24.9.0"
-  },
-  "husky": {
```

Comment:
Updating husky to v7 requires changes in the way we call the hooks, these are now served outside the package.json.

https://github.com/typicode/husky-4-to-7